packages/flutter/lib/src/widgets/_web_browser_detection.dart

```diff
@@ -0,0 +1,11 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ui_web' as ui_web;
+
+/// A class that provides information about the current browser when on the web.
+class WebBrowserDetection {
+  /// Whether the current browser is webkit (Safari).
+  static bool get browserIsSafari => ui_web.BrowserDetection.instance.isSafari;
```

Comment:
Nit: `WebBrowserDetection.browserIsSafari` feels a bit redundant, what do you think of renaming `browserIsSafari` to `isSafari`? I don't feel strongly about this

Reply:
sgtm